README.md

```diff
@@ -28,4 +28,4 @@ implement the ability to filter and sort people in the table.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_people-table-advanced/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://GGLUTT.github.io/react_people-table-advanced/) and add it to the PR description.
```

Comment:
Don't forget to add the demo link to the PR description 
<img width="925" alt="image" src="https://github.com/user-attachments/assets/48ad6dbc-f7a9-44d1-98b1-48eea9dab4c5">